core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -252,6 +252,18 @@ class SparkHadoopUtil extends Logging {
     if (isGlobPath(pattern)) globPath(fs, pattern) else Seq(pattern)
   }
 
+  def expandGlobPath(fs: FileSystem, pattern: Path): Seq[String] = {
+    val arr = pattern.toString.split("/")
```

Comment:
we should not parse the path string ourselves, it's too risky, we may miss some special cases like windows path, escape character, etc. Let's take a look at `org.apache.hadoop.fs.Globber` and see if we can reuse some parser API there.

Reply:
Thanks for your reply, agree with you.